README.md

```diff
@@ -19,52 +19,65 @@ YourAppDelegate.h
 
 YourAppDelegate.m
 ```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    EDQueueStorageEngine *fmdbBasedStorage = [[EDQueueStorageEngine alloc] initWithName:@"database.sample.sqlite"];
+
+    self.persistentTaskQueue = [[EDQueue alloc] initWithPersistentStore:fmdbBasedStorage];
```

Comment:
If we are removing the singleton, then I think we need to include where `self.persistentTaskQueue` is defined.